src/react-components/join-us-dialog.js

```diff
@@ -9,7 +9,7 @@ export default class JoinUsDialog extends Component {
         <span>
           <p>
             Join us in the{" "}
-            <a href="https://discord.gg/wHmY4nd" target="_blank" rel="noopener noreferrer">
+            <a href="https://discord.gg/dFJncWwHun" target="_blank" rel="noopener noreferrer">
```

Comment:
If you're feeling adventurous, you could even remove this entire file. Looks like we don't use it any more.